src/callback/class-run-sniffer-callback.php

```diff
@@ -549,7 +532,25 @@ protected function get_sniff_results( ...$arguments ) {
 
 		$runner->init();
 
+		// Set default standard.
```

Comment:
I was going with this: https://gist.github.com/gsherwood/aafd2c16631a8a872f0c4a23916962ac as a suggestion for a runner. But if moving the config after the init works then it's ok by me :+1: